sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -693,6 +701,15 @@ CLOSE: 'CLOSE';
 
 RENAME: 'RENAME';
 
+REROUTE: 'REROUTE';
+MOVE: 'MOVE';
```

Comment:
please mark those as nonReserved keywords, otherwise we've a breaking change here

Reply:
@mfussenegger good point. I will address that.